docs/getting-started/quick-start.md

```diff
@@ -3,28 +3,68 @@ category: installation
 order: 10
 menu-title: Quick Start
 meta-title: Quick Start | CKEditor 5 documentation
-meta-description: Learn how to install, integrate, configure, and develop CKEditor 5. Browse through the API documentation and online samples.
+meta-description: Learn the fastest way to install a powerful, rich text WYSIWYG editor - CKEditor 5 - in your web application using npm.
 ---
 
-# Getting started with CKEditor&nbsp;5
+# Quick Start
 
-CKEditor&nbsp;5 provides every type of WYSIWYG editing solution imaginable. From editors similar to Google Docs and Medium to Slack or Twitter-like applications, all is possible within a single editing framework. It is an ultra-modern JavaScript rich-text editor with MVC architecture, custom data model, and virtual DOM, written from scratch in TypeScript with excellent webpack and Vite support. Find out the most convenient way to start using it!
+CKEditor&nbsp;5 is a powerful, rich text editor you can embed in your web app. This guide will show you the fastest way to start using it.
```

Comment:
```suggestion
CKEditor&nbsp;5 is a powerful, rich text editor you can embed in your web application. This guide will show you the fastest way to start using it.
```

Reply:
Tbh, I don't know why I used two different notations - I'll change it.